linux_os/guide/system/auditing/auditd_configure_rules/audit_kernel_module_loading/audit_rules_kernel_module_loading_finit/ansible/shared.yml

```diff
@@ -1,4 +1,4 @@
-# platform = multi_platform_rhel
+# platform = multi_platform_rhel, multi_platform_ol
```

Comment:
small nitpick:  this line adds a space whereas others remove it (e.g. ``audit_rules_kernel_module_loading_finit``).

Reply:
@shawndwells, thanks for your review..I agree, looks inconsistent. I noticed spaces in remediation platforms lists handled in different ways (even for RHEL and fedora types). Thus in some cases I was trying to add OL keeping existing approach, while in general preferred to trim values and leave just commas. 
To use spaces or not still confusing stuff for me. Is there any guidance/preference for such lists style to follow?